lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1896,6 +1896,8 @@ void request_initialize(const llvm::json::Object &request) {
   cmd.AddCommand(
       "repl-mode", new ReplModeRequestHandler(),
       "Get or set the repl behavior of lldb-dap evaluation requests.");
+  cmd.AddCommand("custom-event", new CustomDAPEventRequestHandler(),
```

Comment:
what about custom-dap-event?

Reply:
The command is in the `lldb-dap` command group, so I think that would become a bit redundant as `lldb-dap custom-dap-event`, but with [vogelsgesang](https://github.com/vogelsgesang)'s comment below, how about `lldb-dap send-event`?